src/main/routing/topology.c

```diff
@@ -353,7 +359,7 @@ static gboolean _topology_loadGraph(Topology* top, const gchar* graphPath) {
 
     _topology_lockGraph(top);
     info("reading graphml topology graph at '%s'...", graphPath);
```

Comment:
Remove the "graphml" from the info log. (And maybe search to check if other log messages or comments refer to `graphml`).

Reply:
Updated these and also the temporary filename for the graph in controller.c.